theia-ide/theia
# vscode clipboard api + support editor/title for webviews

This PR tries to address some issues of using humao.rest-client vscode extension.

- fix #5519: clipboard vscode api
- fix #5515: support editor/title for webviews

#### How to test

- install humao.rest-client extension
- create test.http and type into some url
- trigger a send request action
- you should be able to toolbar actions
- when you use them the clipboard should be properly populated (at least in Chrome and Electron)

#### Known issues

the active webview is not detected properly right now - https://github.com/theia-ide/theia/issues/5521
because of it toolbar actions for webview does not work always
